scraper/print_collection.py

```diff
@@ -0,0 +1,24 @@
+def print_collection(collection, depth=0, indent=2):
```

Comment:
You can use jsonify or json dumps

http://stackoverflow.com/questions/7907596/json-dumps-vs-flask-jsonify